packages/redux/src/analytics/middlewares/__tests__/bag.test.js

```diff
@@ -9,7 +9,7 @@ import {
   mockBrandId,
   mockCategoryId,
   mockStore,
-} from 'redux/tests';
+} from '../../../../tests';
```

Comment:
Consider configuring an alias for cases like this.

Reply:
There was an alias for that (It was redux/tests) but I removed it because the other packages were doing the same thing as well, so I was trying to be consistent in this case. What do you think?

Reply:
No problem, it was a suggestion just to improve the readability, in my perspective `../../../../tests` seems weird.